### PR TITLE
change confusing note

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6622

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @mypil, for your information

Thank you

File: windows/client-management/troubleshoot-inaccessible-boot-device.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ To verify the BCD entries:
 2. In the **Windows Boot Loader**  that has the **{default}** identifier, make sure that **device** , **path** , **osdevice,**  and **systemroot**  point to the correct device or partition, winload file, OS partition or device, and OS folder.
```
Comment (JohanFreelancer9) [inline windows/client-management/troubleshoot-inaccessible-boot-device.md:113]:
```suggestion
2. In the **Windows Boot Loader**  that has the **{default}** identifier, make sure that **device**, **path**, **osdevice**, and **systemroot**  point to the correct device or partition, winload file, OS partition or device, and OS folder.
```

File: windows/client-management/troubleshoot-inaccessible-boot-device.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ To verify the BCD entries:
...
  
    > [!NOTE]
-   > If the computer is UEFI-based, the **bootmgr** and **winload** entries under **{default}** will contain an **.efi** extension.
+   > If the computer is UEFI-based, the filepath value specified in **path** parameter of **{bootmgr}** and **{default}** will contain an **.efi** extension.
```
Comment (JohanFreelancer9) [inline windows/client-management/troubleshoot-inaccessible-boot-device.md:116]:
```suggestion
   > If the computer is UEFI-based, the filepath value specified in the **path** parameter of **{bootmgr}** and **{default}** will contain an **.efi** extension.
```

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #6622

Thank you.

cc: @kenwith

**Pull request merged**